### PR TITLE
Header implemented by Fetch

### PR DESCRIPTION
**Pull request by Kaleab470**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -155,3 +155,30 @@ function updateMovieText(data) {
 
   return unorderedList;
 }
+
+function displayName(){
+  fetch('/data')
+  .then((response) => response.text()
+
```
Comment (damargulis) [inline portfolio/src/main/webapp/script.js:162]: Remove the commented code and console.log statements

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -155,3 +155,30 @@ function updateMovieText(data) {
...
+    // const introduction = htmlDoc.getElementsByTagName("h1")[0];
+
+    // console.log(introduction.textContent);
+
+    // return introduction.textContent;
+
+    // return response.text();
+  )
+  .then((introduction) => {
+    const parser = new DOMParser();
```
Comment (damargulis) [inline portfolio/src/main/webapp/script.js:177]: In general, it probably wouldn't make sense to structure your api this way -- theres no need to encode the response in html if you are just gonna parse the data out, but you'll learn about JSON in the next lesson which would be the better form to structure data in, so this is fine for now

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Kaleab470) [reply portfolio/src/main/webapp/script.js:162]: Done.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (Kaleab470) [reply portfolio/src/main/webapp/script.js:177]: Okay great!

**Pull request merged**